roles/mistral/defaults/main.yml

```diff
@@ -1 +1,4 @@
 mistral_version: 0.13
+mistral_db_password: StackStorm
```

Comment:
This is not enough, we need also to create database with these credentials in PostgreSQL.
See: https://github.com/StackStorm/ansible-st2/blob/bde33777da7b2327bc0571418bf720ce43f7f716/roles/mistral/meta/main.yml#L20-L24